hwdb.d/60-keyboard.hwdb

```diff
@@ -1689,6 +1689,11 @@ evdev:input:b0003v047FpC006*
 # evdev:atkbd:dmi:bvn*:bvr*:bd*:svnPurism*:pn*Librem13v4*:*
 #  KEYBOARD_KEY_56=backslash
 
+# Purism Librem 11 tablet. 114,115 are the volume keys
+evdev:atkbd:dmi:bvn*:bvr*:svnPurism:pnLibrem11:*
+ WAKEUP_KEY_114=0
+ WAKEUP_KEY_115=0
```

Comment:
Hmm, so we have a mapping of keycodes to key names, but here we use the rew keycodes. This seems iffy. E.g. I can imagine that we'd want to disable wakeup using VOLUME_UP and VOLUME_DOWN keys across a broad range of devices, even if the numerical keycodes are different. Shouldn't the wakeup listing use key names instead?

Reply:
Indeed was thinking about that too when reworking this. The upside of the keycodes is that they're independent from any known list of key names.

Reply:
FWIW this is how I originally imagined it: https://gitlab.gnome.org/World/Phosh/phoc/-/issues/363#note_2069042

When I got to the code, though, I discovered that there isn't really a cohesive / libbified view of the linux events UAPI in the reverse direction. (i.e key names to keycodes).

For example, evtest does this to show the key names: https://gitlab.freedesktop.org/libevdev/evtest/-/blob/master/evtest.c?ref_type=heads#L223

So maybe this isn't necessarily a good reason but I'll try anyway ;) I think keycodes are preferable here because ultimately, what's coming out of the keyboard *is* just a keycode, and it reduces the burden on consumers of this information.

Reply:
> E.g. I can imagine that we'd want to disable wakeup using VOLUME_UP and VOLUME_DOWN keys across a broad range of devices, even if the numerical keycodes are different.

I'm also not sure how this would work in practice? We're not in XKB land at this point, so [`KEY _VOLUME_UP`](https://github.com/torvalds/linux/blob/master/include/uapi/linux/input-event-codes.h#L190) is always just `115`

Reply:
I think there's a bit of confusion on scancode (hardware) vs keycode (evdev) vs keysym (xkb). we can discount keysyms since they don't exist at this level.

So it's now down to scancode (like `KEYBOARD_KEY_xx`) which are hardware-specific or evdev keycodes. The latter is a lot more flexible so we should go with that but for that I would a) move this to a separate hwdb file to avoid confusion and b) use an `EVDEV_` prefix to make this clear. We already use `EVDEV_ABS_00` for `ABS_X`, would be fine to use a hex `EVDEV_KEY_xx` imo.

A separate hwdb is IMO better if we have device-wide state properties like the `WAKEUP_KEY_DEFAULT` too.

Reply:
> The latter is a lot more flexible so we should go with that but for that

I hope there's no confusion in the actual patch which uses keycodes? 

A separate file sounds good to me. I assume it's fine to reuse `rules.d/60-evdev.rules` ?